cypress/e2e/buy-shirt.cy.ts

```diff
@@ -1,20 +1,24 @@
 /// <reference types ="Cypress" />
 
-import { LoginPage } from "../pages/index";
+import { LoginPage, ProductsContentPage} from "../pages/index";
 
 const loginPage = new LoginPage();
+const productPage = new ProductsContentPage();
 
 describe("Buy a product workflow", () => {
   describe("Given the user logs in with a valid and active user", () => {
     beforeEach(() => {
       loginPage.visitLoginPage();
       loginPage.signIn("standard_user", "secret_sauce");
+      productPage.displayContainer();
```

Comment:
asserts should happen only inside `it` fn, also displayContainer seems weird, usually you can read on the test what you are testing, maybe a new test need to be addded here?